File: 351001/Rudzenia/RW.Application/DTOs/Response/AuthorResponseTo.cs
using System.Text.Json.Serialization;
using RW.Domain.Entities;

namespace RW.Application.DTOs.Response;

public class AuthorResponseTo
{
    [JsonPropertyName("id")]
    public long Id { get; set; }

    [JsonPropertyName("login")]
    public string Login { get; set; } = string.Empty;

    [JsonPropertyName("password")]
    public string Password { get; set; } = string.Empty;

    [JsonPropertyName("firstname")]
    public string FirstName { get; set; } = string.Empty;

    [JsonPropertyName("lastname")]
    public string LastName { get; set; } = string.Empty;

    [JsonPropertyName("role")]
    [JsonConverter(typeof(JsonStringEnumConverter))]
    public Role Role { get; set; } = Role.CUSTOMER;
}

File: 351004/Romanovskiy/task330/publisher/src/main/java/com/example/task330/domain/dto/response/AuthorResponseTo.java
package com.example.task330.domain.dto.response;

import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.AllArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class AuthorResponseTo {
    private Long id;
    private String login;
    private String firstname;
    private String lastname;
}